winter/pagination/limits.py

```diff
@@ -0,0 +1,12 @@
+import dataclasses
+
+
+@dataclasses.dataclass(frozen=True)
+class LimitsAnnotation:
+    default: int
+    maximum: int
```

Comment:
Consider using None value to describe "No limit".

Reply:
yep